### PR TITLE
add react select (originally from josephdlawson21-add-react-select). Fixes #892

### LINKED ISSUE #892: Spouse/member dropdown not fully changing on selection of item
### Replicate:
1. Go to the 'Household' section
2. Add a member
3. Click on the drop down
4. Click on 'Child/Other member'
5. Click on the drop down again
6. Click off of the drop down

### Result I'd Expect:
- 'Child/Other member' to appear as the chosen option

### Actual Result:
- 'Spouse' is the text that appears as the selected option

### Environment:

App version or commit id at the time: 04d75ab296d8648456fdb4fecc5bb84be241f64f

### PR DESCRIPTION
**Pull request by turnerhayes**
No idea why this is passing CI and the original PR (#977) was not...

### CODE REVIEW

**Comment by turnerhayes** [comment]
Fixes #892.

**Comment by knod** [comment]
It's because the package-lock wasn't updated, I think.

**Comment by knod** [comment]
We'll have to try to fix the styles on here after merging this.

**Review by knod — APPROVED** [review]
Looks good! Though we gotta fix those styles.

**Comment by turnerhayes** [comment]
@knod Did a merge that changed Household.js--would appreciate a review of the new merge!

**Comment by turnerhayes** [comment]
@dylanesque (or anybody else)--I would like to have at least 2 approvals on this before I merge, since I didn't write the original.

File: src/forms/Household.js
Comment (knod) [inline src/forms/Household.js:230]: This and the surrounding changes don't quite match the style from above, but I'm not too bothered. If you're interested in changing it, you'd change this line to:
```js
setDropdownMember = { onMemberDropdownChange }
```
and then adjust the other lines to match.

**Comment by knod** [comment]
Doesn't look like it broke anything to me and the code styles are pretty close to what we're establishing. I stand by my approval!

**Review by dylanesque — APPROVED** [review]

**Pull request merged**
